scripts/aot.spec.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+const shell = require('shelljs');
+const VERSION = require('../package.json').version;
+const colors = require('colors/safe');
+
+const packagePath = `../../dist/clr-angular/clr-angular-${VERSION}.tgz`;
+const installCommand = `npm i ${packagePath} --save`;
+
+// Moving to the ks-app root to work.
+console.log(colors.yellow('Testing Clarity Artifacts for AOT'));
+process.chdir('./src/ks-app');
+
+// Do package installation.
+console.log(colors.yellow(`Installing latest @clr/angular with: ${packagePath}`));
+shell.exec(installCommand);
+shell.exec('npm i');
+
+// Compile ks-app for production.
+// NOTE: this must stay here (after the script cd's into src/ks-app)!!
+let status = shell.exec('npm run build --preserve-symlinks');
+console.log(colors.yellow('Testing Clarity AOT on ks-app'));
+if (status.code === 0) {
+    console.log(colors.green('Clarity passed AOT compilation in the ks-app.'));
+} else {
+    console.log(colors.red(`AOT compilation failed on ks-app!! Status: ${status.code}\n`));
```

Comment:
I suspect we should explicitly throw an error or `process.exit(1)` to ensure its caught.

Reply:
Didn't see `process.exit` in my auto-completion list. Will revisit.